content/uiux/concepts/bias-in-user-research/bias-in-user-research.md

```diff
@@ -0,0 +1,111 @@
+---
+Title: 'Bias in User Research'
+Description: 'In user research, bias refers to prejudices or tendencies in researchers’ and participants’ ways of thinking that can lead to inaccurate or skewed results.'
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+In [user research](https://www.codecademy.com/resources/docs/uiux/user-research), __bias__ refers to prejudices or tendencies in researchers’ and participants’ ways of thinking that can lead to inaccurate or skewed results. 
+
+To obtain accurate, useful data, researchers must be aware of their own biases. Researchers can also work to prevent participants from displaying common biases by strategically designing their studies. When analyzing results, researchers should consider how bias may have impacted the data, especially if the data is self-reported. 
+
+### Acquiescence Bias 
+
+ __Acquiescence bias__ is the tendency to agree with the interviewer. In general, participants are more likely to agree than to disagree. Thus, the question should not imply that there is a "correct" answer. Instead of asking for agreement or disagreement to a predefined statement, researchers can ask users to explain their point of view.  
+
+<details open> 
+
+<summary>Example</summary> 
+
+<br> 
+
+*“Would you agree that you are satisfied with the current interface?”* This question implies that the user is satisfied with the interface and could lead to acquiescence bias. <br>    
+
+*“How do you feel about the current interface?”* This phrasing is open-ended, so it does not lead participants to agree with a particular sentiment. </details> 
```

Comment:
[Proofreading] Remember to put backticks when using code snippets. 
```suggestion
```html
<details open> 

<summary>Example</summary> 

<br> 

*“Would you agree that you are satisfied with the current interface?”* This question implies that the user is satisfied with the interface and could lead to acquiescence bias. <br>    

*“How do you feel about the current interface?”* This phrasing is open-ended, so it does not lead participants to agree with a particular sentiment. </details>
``` 
```